src/network/core/os_abstraction.h

```diff
@@ -83,6 +83,15 @@ typedef unsigned long in_addr_t;
 #	include <errno.h>
 #	include <sys/time.h>
 #	include <netdb.h>
+
+#   if defined(__EMSCRIPTEN__)
+/* Emscripten doesn't support AI_ADDRCONFIG and errors out on it. */
+#		undef AI_ADDRCONFIG
+#		define AI_ADDRCONFIG 0
+/* Emscripten says it supports FD_SETSIZE fds, but it really only supports 64. */
+#		undef FD_SETSIZE
+#		define FD_SETSIZE 64
```

Comment:
In 2013 somewhere they said they wanted to fix this .. they never did. So I wouldn't get my hopes up this will ever be fixed upstream.

Reply:
Again, would be good to have a link to the (closed) issue for reference